server/pulp/server/managers/repo/publish.py

```diff
@@ -165,7 +165,9 @@ def _do_publish(repo, distributor_id, distributor_instance, transfer_repo, condu
             result_code = RepoPublishResult.RESULT_SUCCESS
 
         result = RepoPublishResult.expected_result(
-            repo_id, repo_distributor['id'], repo_distributor['distributor_type_id'],
+            repo_id, repo_distributor['id'],
+            call_config.flatten(),
+            repo_distributor['distributor_type_id'], call_config.flatten(),
             publish_start_timestamp, publish_end_timestamp, summary, details, result_code)
```

Comment:
Please make sure that the above is explicitly tested so we can be sure that it works properly.